component-generator-ui/src/utils/Constants.js

```diff
@@ -0,0 +1,44 @@
+import { v4 as uuidv4 } from 'uuid';
+import { randomId } from './Utils';
+
+export const FORM_TYPES = [
+    { value: 'checkbox', label: 'Checkbox' },
+    { value: 'datepicker', label: 'Date picker' },
+    { value: 'heading', label: 'Heading (for decorative help text)' },
+    { value: 'hidden', label: 'Hidden field' },
+    { value: 'image', label: 'Image field' },
+    { value: 'multifield', label: 'Multifield' },
+    { value: 'numberfield', label: 'Number field' },
+    { value: 'pagefield', label: 'Page Field' },
+    { value: 'pathfield', label: 'Path field' },
+    { value: 'radio', label: 'Radio field (Child property)' },
+    { value: 'radiogroup', label: 'Radio group' },
+    { value: 'select', label: 'Select box (Granite select)' },
+    { value: 'tagfield', label: 'Tag Field' },
+    { value: 'textarea', label: 'Text area' },
+    { value: 'textfield', label: 'Text field' },
+];
+export const SLING_ADAPTABLES = [
+    { value: 'resource', label: 'Resource' },
+    { value: 'request', label: 'SlingHttpServletRequest' },
+];
+export const EMPTY_PROP = {
+    field: '',
+    description: '',
+    javadoc: '',
+    type: '',
+    label: '',
+    jsonExpose: false,
+    useExistingModel: false,
+    id: uuidv4(),
+    attributes: [],
+    items: [],
+};
+export const EMPTY_TAB = {
+    label: '',
+    id: `tab-${randomId(5)}`,
```

Comment:
Mentioned in another place - do we need these to be random, or can we simply use `1`, `2`, `3` to keep the tab node names clean and in order?